config/routes.rb

```diff
@@ -38,6 +38,10 @@
       resources :forms, only: [:new, :create], module: :hired
     end
 
+    namespace :businesses do
+      resources :hiring_invoice_requests, only: [:new, :create]
+    end
+
```

Comment:
Let's move this to line 17 so it goes "right after" `resources businesses`.

Reply:
moved in efa30812efc2df53eda3669f32ed8a7c01127f7b